client/src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Redirect, Route, RouteProps } from 'react-router';
+import { useAuth } from '../../context/useAuthContext';
+
+export interface IPrivateRouteProps extends RouteProps {
```

Comment:
```suggestion
export interface PrivateRouteProps extends RouteProps {
```
It is recommended not to prefix interfaces with 'I' in JavaScript

Reply:
sure , done